src/main/const.ts

```diff
@@ -13,8 +13,12 @@ const kangarooConfigString = fs.readFileSync(
 );
 export const KANGAROO_CONFIG: KangarooConfig = JSON.parse(kangarooConfigString);
 
-export const DEFAULT_BOOTSTRAP_SERVER = 'https://bootstrap.holo.host';
-export const DEFAULT_SIGNALING_SERVER = 'wss://sbd.holo.host';
+export const DEFAULT_BOOTSTRAP_SERVER = 'https://dev-test-bootstrap2.holochain.org';
```

Comment:
Should we be recommending these here? If we're saying these really are test services then we need to treat them as such and not encourage app developers to use them. I think we want launcher to use them by default so that people get an "out of the box" experience but for Kangaroo I think we should consider whether this is appropriate

Reply:
Well, I was thinking that we're still recommending the main-0.4 branch so this is just until we have the proper production servers for 0.5 and then we'd change that here.

Reply:
I'm fine with that.